### DISCUSSION TITLE
X96 Mini W2 support ?

### DISCUSSION THREAD

**Discussion by rozhkoserhii** [devmfc/debian-on-amlogic#71] [Q&A] [answered]
Tell me, do X96 Mini W2 images based on Amlogic S905W2 support them? I plan to take it, but I didn’t find it in the list of supported devices, only the processor is similar.

**Comment by devmfc** [comment, marked as answer]
See https://github.com/devmfc/debian-on-amlogic/discussions/65#:~:text=Settings-,%5Bx96mini%20s905W2%5D%20Kernel%20panic,-%2365.

But if you want to have a working server, I would advice against buying one of the newer SOCs because mainline support is not ready yet.

**Reply by rozhkoserhii** [reply]
Thank you for your response. I wish you success with your project
